tools/topology/sof-apl-nocodec.m4

```diff
@@ -237,43 +237,43 @@ DAI_CONFIG(SSP, 0, 0, NoCodec-0,
 		      SSP_CLOCK(bclk, 1536000, codec_slave),
 		      SSP_CLOCK(fsync, 48000, codec_slave),
 		      SSP_TDM(2, 16, 3, 3),
-		      dnl SSP_CONFIG_DATA(type, dai_index, valid bits, mclk_id)
-		      SSP_CONFIG_DATA(SSP, 0, 16)))
+		      dnl SSP_CONFIG_DATA(type, dai_index, valid bits, mclk_id, quirks)
+		      SSP_CONFIG_DATA(SSP, 0, 16, 0, 64)))
```

Comment:
64 is a magic number, we can define it as a LBM mode macro in ssp.m4.

Reply:
sure i will define a flag in ssp.m4